.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: java
 jdk:
-  - openjdk7
+  - openjdk8
   - oraclejdk8
   - oraclejdk9
```

Comment:
Shouldn't oraclejdk10 be listed here too (it is referenced in README.md below)?

Reply:
It should but Travis hasn't got support form it yet.